embedded_resources/web_interface/index.js

```diff
@@ -572,31 +581,29 @@ $(".act-reboot").addEventListener("click", async (e) => {
   }, 1000);
 });
 
-window.addEventListener("keydown", (e) => {
+window.addEventListener("keydown", async (e) => {
   let key = e.key.toLowerCase()
-  if (e.target.matches(".file-content") || $(".dialog.editor").classList.contains("hidden") === false) {
-    if ((e.ctrlKey || e.metaKey) && !e.shiftKey && key === "s") {
+  if ($(".dialog.editor:not(.hidden)")) { // means editor tab is open
+    if ((e.ctrlKey || e.metaKey) && key === "s") {
       e.preventDefault();
       e.stopImmediatePropagation();
-      saveFile();
-    }
-    if ((e.ctrlKey || e.metaKey) && e.shiftKey && key === "s") {
+
+      await saveEditorFile();
+    } else if ((e.altKey || e.metaKey) && key === "enter") {

```

Comment:
`|| e.metaKey` is not required, unless you meant for Command + Enter to work on Macs as well as Alt.

Reply:
yep, it's intended, since there's no alt button in mac.

Reply:
While there is no Alt key, e.altKey is the Option button on Mac.

Different browsers, different keyboards, so much complication just to get a simple keyboard shortcut to work everywhere!